charts/opensearch-dashboards/templates/deployment.yaml

```diff
@@ -151,6 +154,18 @@ spec:
         - containerPort: {{ .Values.service.port }}
           name: {{ .Values.service.httpPortName | default "http" }}
           protocol: TCP
+{{- if .Values.plugins.enabled }}
+        command:
+          - sh
+          - -c
+          - |
+            #!/usr/bin/bash
+            set -e
+            {{- range $plugin := .Values.plugins.installList }}
+            ./bin/opensearch-dashboards-plugin install {{ $plugin }}
+            {{- end }}
```

Comment:
Hey @BionIT quick question, for OpenSearch we have a `--batch` option to force install without a prompt, is there a same flag or scenario for `opensearch-dashboards-plugin`, I dont see it in the document though https://opensearch.org/docs/latest/dashboards/install/plugins/, please test this and share your update.

Reply:
Hey @prudhvigodithi, --batch is not supported by opensearch-dashboards-plugin(`error: unknown option '--batch'
`), I have been using this script to install plugins and never had prompts.